tools/analyzer_plugin/lib/src/assist/refs/add_create_ref.dart

```diff
@@ -87,8 +87,9 @@ void addUseOrCreateRef(
   }
 
   if (refTypeToReplace == RefTypeToReplace.callback) {
+    if (createRefField == null) return;
```

Comment:
This null-check was added since the first arg to `lookUpVariable` was made non-nullable.